target/linux/ath79/dts/qca9533_glinet_ar300m.dts

```diff
@@ -0,0 +1,115 @@
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	compatible = "glinet,ar300m", "qca,qca9557";
```

Comment:
this should be 9553

Reply:
Ah, I did this backwards. I thought I had put in that, but was using 9557's dtsi, not both
as 9557. Will fix up, thanks.

Reply:
from wikidevi I see that it uses QCA9531, I made a template for 9533 in wr841n-v9 (mostly a qca9531), maybe you can use it as a reference and after that qca9533 to use it as a reference too, is your clock detected correctly using the `qca9557.dtsi` as a template ?

Reply:
@lucize looks fine, yes. which clock do you mean, btw?

Reply:
cpu mhz, but if the board uses ref clock 40mhz then you are ok

Reply:
Its a little off from what is given during an ar71xx bootup, but its close enough. I don't see
a branch on your repo for wr841n-v9, btw.

Reply:
it should be exactly like original, else the time will drift
https://github.com/openwrt/openwrt/pull/956

Reply:
Double checking shows the cpu clock is correct.